main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
Ah, I didn't think of creation, but don't you get it for free? I.e. if the ground truth is in the attribute and the special setters and getters just help you manipulate the attribute, then wouldn't it be okay?